statistics/handle/handle_hist.go

```diff
@@ -369,7 +369,10 @@ func (h *Handle) finishWorking(col model.TableColumnID) {
 	defer h.StatsLoad.Unlock()
 	if chList, ok := h.StatsLoad.workingColMap[col]; ok {
 		for _, ch := range chList {
-			ch <- col
+			select {
+			case ch <- col:
+			default:
+			}
```

Comment:
It will still panic if the ch is closed by another goroutine:
ch := make(chan string)
	close(ch)
	go func() {
		select {
		case ch <- "a":
		default:
		}
	}()
panic: send on closed channel

Reply:
It is my fault. Fixed.